### PR TITLE
Std : add Require js doc

### PR DESCRIPTION
**Pull request by zekth**
Adding require js doc for std
It's just a start, i'd like to have some help for this, because i'm not aware of all the modules in the lib.

EDIT:
Third party repo needs to be updated because of the addition of jsdoc plugin: https://github.com/denoland/deno_third_party

### CODE REVIEW

File: std/util/deep_assign.ts [hunk 1]
```diff
@@ -1,4 +1,9 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+/**
+ * Do a deep assignation of Objects
```
Comment (nayeemrmn) [inline std/util/deep_assign.ts]: assignment?

Comment (zekth) [reply std/util/deep_assign.ts]: Sorry non native english so there's maybe typos.

File: std/installer/mod.ts [hunk 2]
```diff
@@ -195,6 +198,13 @@ exit $ret
   await chmod(filePath, 0o755);
 }
 
+/**
+ * Install modules
+ * @param moduleName
```
Comment (kitsonk) [inline std/installer/mod.ts]: If the plugin doesn't allow omission of useless `@param` without any descriptions I would be 👎 for this.  This is just vertical noise in files of no value.

Comment (zekth) [reply std/installer/mod.ts]: It can be tuned as shown here : https://www.npmjs.com/package/eslint-plugin-jsdoc#require-param-description

**Comment by ry** [comment]
Closing because red and old.

**Comment by zekth** [comment]
Old? ` 9 days ago` and requires this one :
https://github.com/denoland/deno_third_party/pull/52

Seriously...

**Comment by ry** [comment]
@zekth Sorry - no offense - I just don't want to have red PRs sitting around for many days - it makes it more difficult for me on the project management side to have to revisit PRs. It doesn't seem like this is close to landing either. We need to start by creating a list missing docs, then work through those, then add this lint change.

**Pull request closed without merging**